indigo_app/urls.py

```diff
@@ -36,11 +36,14 @@
     re_path(r'^places/(?P<place>[a-z]{2}(-[^/]+)?)/tasks/(?P<pk>\d+)/assign', tasks.TaskAssignView.as_view(), name='assign_task'),
     re_path(r'^places/(?P<place>[a-z]{2}(-[^/]+)?)/tasks/(?P<pk>\d+)/unassign', tasks.TaskAssignView.as_view(unassign=True), name='unassign_task'),
     re_path(r'^places/(?P<place>[a-z]{2}(-[^/]+)?)/tasks/(?P<pk>\d+)/projects', tasks.TaskChangeWorkflowsView.as_view(), name='task_workflows'),
+    re_path(r'^places/(?P<place>[a-z]{2}(-[^/]+)?)/tasks/(?P<pk>\d+)/block_by_task', tasks.TaskChangeBlockingTasksView.as_view(), name='task_blocked_by'),
```

Comment:
Perhaps `.../blocking-tasks`? I prefer dashes over underscores in URLs.

Reply:
I used a verb to follow the other patterns;  what do you think of `change-blocking-tasks`? that would also correspond with the view name

Reply:
It's a mixed bag. If we're doing REST (which we vaguely are, eg. projects), then the endpoints should actually be nouns and the HTTP verb determines the action. This doesn't work well for assign, unassign, etc. But it does kinda work for blocking tasks -- you're POSTing to the blocking-tasks collection.

If you're strictly using RPC-style naming, like a method name, then it would be a verb as you say.